design/raft.md

```diff
@@ -51,9 +51,20 @@ node's certificate, but the Raft ID is assigned arbitrarily and would change.
 
 It's important to note that a Raft ID can't be reused after a node that was
 using the ID leaves the consensus group. These Raft IDs of nodes that are no
-longer part of the cluster are saved in a list to make sure they aren't reused.
-If a node with a Raft ID on this list tries to use Raft RPCs, other nodes won't
-honor these requests.
+longer part of the cluster are saved (snapshot persisted) in a list (a blacklist,
```

Comment:
Minor nitpick - it's saved as part of the raft logs, which may be a combination of a snapshot and the WAL, so possibly just saying that they're saved in raft would be clearer?